generators/server/templates/_pom.xml

```diff
@@ -1203,37 +1203,12 @@
             </dependencies>
             <build>
                 <plugins>
-                    <%_ if (!skipClient) { _%>
+                    <%_ if (!skipClient && clientFw === 'angular2') { _%>
                     <plugin>
                         <groupId>com.github.eirslett</groupId>
                         <artifactId>frontend-maven-plugin</artifactId>
                         <version>${frontend-maven-plugin.version}</version>
                         <executions>
-                            <%_ if (yarnInstall) { _%>
```

Comment:
@deepu105 : maybe I drop too much code here! We should support yarn too for ng2 of course, but I forget to ask you this question: in dev profile, is it mandatory to npm install / yarn for ng2 ?